src/api/app/policies/project_policy.rb

```diff
@@ -17,4 +17,8 @@ def update?
   def destroy?
     update?
   end
+
+  def new_incident?
```

Comment:
Do you expect that the new_incident policy and the update policy will diverge in the future?